boards/arm/stm32h747i_disco/doc/index.rst

```diff
@@ -63,6 +63,15 @@ The current Zephyr stm32h747i_disco board configuration supports the following h
 +-----------+------------+-------------------------------------+
 | GPIO      | on-chip    | gpio                                |
 +-----------+------------+-------------------------------------+
+| ETHERNET  | on-chip    | ethernet  (*)                       |
++-----------+------------+-------------------------------------+
+| RNG       | on-chip    | True Random number generator        |
```

Comment:
Each peripheral enablement should have its own commit.

Reply:
Splitted as suggested.